.circleci/config.yml

```diff
@@ -272,6 +272,24 @@ jobs:
             conda create -y --name pyveloxenv python=3.7
             conda activate pyveloxenv
             LD_LIBRARY_PATH=/usr/local/lib make python-test
+      - run:
+          name: "Check function signatures"
+          command: |
+            source ~/.bashrc
```

Comment:
why do we need this?

Reply:
Without this conda might fail and not activate the environment. Conda writes its activation code in <shell>rc file and this is just to ensure its loaded.